server/src/main/java/io/druid/client/ServerViewUtil.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.client;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import io.druid.client.selector.ServerSelector;
+import io.druid.query.DataSource;
+import io.druid.query.LocatedSegmentDescriptor;
+import io.druid.query.SegmentDescriptor;
+import io.druid.query.TableDataSource;
+import io.druid.server.coordination.DruidServerMetadata;
+import io.druid.timeline.TimelineLookup;
+import io.druid.timeline.TimelineObjectHolder;
+import io.druid.timeline.partition.PartitionChunk;
+import org.joda.time.Interval;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ */
+public class ServerViewUtil
+{
+  public static final int DEFAULT_NUM_CANDIDATES = 5;
```

Comment:
minor: @navis someone my want to have all the server by default but maybe be you have a specific use case in mind ?